.github/workflows/test.yml

```diff
@@ -344,8 +344,9 @@ jobs:
       - name: Test ssh connection from runner into ubuntu user
         run: ssh -i $HOME/.ssh/id_ed25519 -o PreferredAuthentications=publickey -o PasswordAuthentication=no -l ubuntu ${{ steps.test_container_ipv4.outputs.IPv4 }} uptime
 
-      - name: Test ssh connection into root user with password authentication
-        run: sshpass -p 'password' ssh -i /dev/null -l ubuntu ${{ steps.test_container_ipv4.outputs.IPv4 }} uptime
+      # Default SSH config is PermitRootLogin prohibit-password
```

Comment:
We could tweak the configuration here, but I'm not sure root password login is something we should test or encourage. It doesn't seem to break anything in the workflow, at least.

Reply:
I don't like this. There are more tests for password auth which were added over time, specifically to catch problems when ssh is setup the wrong way.

Reply:
IMHO, `PermitRootLogin yes` and `PasswordAuthentication yes` **is** the wrong setup, but I've added it back.

Reply:
Oh, I agree that this is not something which is best practice. Nevertheless if someone is using root and passwords in their setup, the plugin better works. You can't enforce not to use this setup, therefore better have it tested.